instrumentation/aerospike-client/aerospike-client-7.1/library/src/test/java/io/opentelemetry/instrumentation/aerospike/v7_1/AerospikeMetricsTest.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.aerospike.v7_1;
+
+import static io.opentelemetry.instrumentation.aerospike.v7_1.AerospikeSemanticAttributes.AEROSPIKE_ERROR_CODE;
+import static io.opentelemetry.instrumentation.aerospike.v7_1.AerospikeSemanticAttributes.AEROSPIKE_NAMESPACE;
+import static io.opentelemetry.instrumentation.aerospike.v7_1.AerospikeSemanticAttributes.AEROSPIKE_SET_NAME;
+import static io.opentelemetry.instrumentation.aerospike.v7_1.AerospikeSemanticAttributes.AEROSPIKE_STATUS;
+import static io.opentelemetry.instrumentation.aerospike.v7_1.AerospikeSemanticAttributes.AEROSPIKE_USER_KEY;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.OperationListener;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.testing.exporter.InMemoryMetricReader;
+import io.opentelemetry.semconv.SemanticAttributes;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Test;
+
+class AerospikeMetricsTest {
+
+  @Test
+  @SuppressWarnings("deprecation")
+  // until old http semconv are dropped in 2.0
+  void collectsMetrics() {
+    InMemoryMetricReader metricReader = InMemoryMetricReader.create();
```

Comment:
Testing metrics this way is a bit unusual, it would be easier to combine traces test with metrics like in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/30ddf6a66cca10f1f845b83c3f8a35c0c797edaa/testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpClientTest.java#L709

Reply:
Yeah, indeed. Merged metrics tests with traces tests.